content/guides/collaborations/connect-slack-to-group-collabs/0-index.md

```diff
@@ -0,0 +1,81 @@
+---
+rank: 0
+type: quick-start
+hide_step_number: true
+hide_in_page_nav: true
+icon: FiUsers
+---
+
+# Connect Slack to Box Group Collaborations
```

Comment:
Would it be ok to explore some alternative titles? I'd like to explore if we can make it clearer what this does. Connect is a loose word, so maybe we can say something like sync? I also think it's worth mentioning the term channel in the title, as that's what we're syncing?

Ideas:
* Connect Slack channels to Box groups
* Sync Slack channel users to Box groups

Reply:
FYI this was me. Swithed account now.